FabLabKasse/faucardPayment/MagPosLog.py

```diff
@@ -34,13 +34,13 @@ def __init__(self, amount, cur, con):
         self.payed = False
         self.cur = cur
         self.con = con
-        self.amount = float(amount)
+        self.amount = amount
         self.timestamp_payed = None
         self.oldbalance = 0
         self.newbalance = 0
 
         # Create table if it does not exist
-        self.cur.execute("CREATE TABLE IF NOT EXISTS MagPosLog(id INTEGER PRIMARY KEY AUTOINCREMENT, datum, cardnumber INT, amount FLOAT, oldbalance INT, newbalance INT, timestamp_payed, status INT, info INT, payed)")
+        self.cur.execute("CREATE TABLE IF NOT EXISTS MagPosLog(id INTEGER PRIMARY KEY AUTOINCREMENT, datum, cardnumber INT, amount TEXT, oldbalance INT, newbalance INT, timestamp_payed, status INT, info INT, payed)")
```

Comment:
The problem is that the database does not store it with unlimited precision as far as i noticed. The amount 2.04999 were therefore cut to be 2.04 ...
The conversion was already done this evening (SQLITE can do that on its own) but not yet pushed to Kasse. I verified the conversion by summing the columns and comparing side by side and have not yet seen any difference in data so far.

The column type TEXT was choosen to copy the structure of kassenbuch, where "betrag" is of type TEXT.